README.md

```diff
@@ -79,21 +79,42 @@ database according to the given profile before execute the test suit.
            
     * MSSQL CTP 2.0:
     
-            mvn verify -P local-mssql -f /component/pom.xml -Dskip.surefire.test=true
+            mvn verify -P local-mssql -f /component/pom.xml -Dskip.surefire.test=true -Ddocker.removeVolumes=true
             
     * Oracle 11.2.0.2 Express Edition:
```

Comment:
Comment says Oracle 11

Reply:
and line 86 its mentioned `local-oracle12`

Reply:
Fixed in 09e6cb3